spot_hardware_interface/include/spot_hardware_interface/spot_hardware_interface.hpp

```diff
@@ -140,6 +149,7 @@ class SpotHardware : public hardware_interface::SystemInterface {
   // The 3 state interfaces are position, velocity, and effort.
   static constexpr size_t state_interfaces_per_joint_ = 3;
   size_t njoints_;
+  size_t nfeet = 4;
```

Comment:
```suggestion
  static constexpr size_t kNumFeet_ = 4;
```
seems like this is a constant?